keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifact.kt

```diff
@@ -12,8 +10,9 @@ data class DebianArtifact(
   override val deliveryConfigName: String? = null,
   override val reference: String = name,
   val vmOptions: VirtualMachineOptions,
-  val statuses: Set<ArtifactStatus> = emptySet(),
+  override val statuses: Set<ArtifactStatus> = emptySet(),
   override val versioningStrategy: VersioningStrategy = DebianSemVerVersioningStrategy
 ) : DeliveryArtifact() {
-  override val type = ArtifactType.deb
+  override val type = DEBIAN
+  override fun toString(): String = super.toString()
```

Comment:
isn't this the default behavior?

Reply:
That's what I thought, but it turns out `data class` overrides `toString`.